tests/s2n_test.h

```diff
@@ -262,6 +262,8 @@ void s2n_test__fuzz_cleanup() \
 } \
 int LLVMFuzzerInitialize(int *argc, char **argv[]) \
 { \
+    test_count = 0; \
+    EXPECT_SUCCESS_WITHOUT_COUNT(s2n_in_unit_test_set(true)); \
```

Comment:
Do we want the fuzzer to act like a unit test? Don't we want it to be more like production?

Reply:
The way the fuzzers are currently written they act more like unit tests (e.g. calling private APIs, inspecting internal state, etc). I think if you want a more production-like fuzzer, it would be better to do an end-to-end fuzzer that tests the whole thing together.

Reply:
removed this